packages/cactus-plugin-ledger-connector-besu/src/main/json/openapi.json

```diff
@@ -816,6 +816,85 @@
                     }
                 }
             },
+            "DeployContractSolidityBytecodeJsonObjectV1Request": {
+                "type": "object",
+                "required": [
+                    "contractName",
+                    "contractAbi",
+                    "bytecode",
+                    "web3SigningCredential",
+                    "contractJSON",
+                    "constructorArgs"
+                ],
+                "additionalProperties": false,
+                "properties": {
+                    "contractName": {
+                        "type": "string",
+                        "description": "The contract name for retrieve the contracts json on the keychain.",
+                        "minLength": 1,
+                        "maxLength": 100,
+                        "nullable": false
+                    },
+                    "contractAbi": {
```

Comment:
@TonyRowntree Is `contractAbi` being used anywhere in the plugin's method that uses this request object? I couldn't find it.